hist/hist/inc/TF1.h

```diff
@@ -21,17 +21,17 @@
 //                                                                      //
 //////////////////////////////////////////////////////////////////////////
 
-#include "RConfigure.h"
```

Comment:
It's only used to check for `R__HAS_VECCORE`, which is provided by `Math/Math.h`.